juntagrico/tests/test_jobs.py

```diff
@@ -192,7 +193,6 @@ def testAssignmentEditByCoordinator(self):
         self.testAssignmentEdit(self.area_admin)
         self.testAssignmentDelete(self.area_admin)
 
-
```

Comment:
Please keep this second newline, according to the python style guide. https://peps.python.org/pep-0008/
The branch releases/2.0 will enforce it.